components/Tag/index.js

```diff
@@ -7,8 +7,8 @@ export default function Tag({ tag, foundTag, editTag, activityId, tagId }) {
   function handleClick(event) {
     editTag(activityId, tagId);
     setOpacity(opacity === 1 ? 0.5 : 1);
+    console.log("edittag and activityid", activityId, tagId);
```

Comment:
This console.log is not needed for your feature, but was used during your developing process and should therefore be removed.